src/autofill-utils.js

```diff
@@ -321,11 +321,13 @@ const isLikelyASubmitButton = (el) => {
     const ariaLabel = el.getAttribute('aria-label') || ''
     const title = el.title || ''
     const value = (el instanceof HTMLInputElement ? el.value || '' : '')
+    const dataTestId = el.getAttribute('data-test-id') || ''
```

Comment:
For context, this is the websites own test IDs, right? i.e. This isn't our own testing attributes leaking into actual code